jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -203,6 +220,7 @@ data class CodeModernizerSessionContext(
                     )
                     return null
                 } else {
+                    shouldTryMvnCommand = false
                     LOG.warn { "Maven executed successfully" }
```

Comment:
Set `shouldTryMvnCommand` as false when mvn build succeeds to skip the IDE bundled maven build.